bindings/matrix-sdk-crypto-nodejs/src/requests.rs

```diff
@@ -243,8 +253,8 @@ macro_rules! request {
 }
 
 request!(KeysUploadRequest from RumaKeysUploadRequest maps fields device_keys, one_time_keys, fallback_keys);
-request!(KeysQueryRequest from RumaKeysQueryRequest maps fields timeout, device_keys, token);
-request!(KeysClaimRequest from RumaKeysClaimRequest maps fields timeout, one_time_keys);
+request!(KeysQueryRequest from RumaKeysQueryRequest maps fields timeout { timeout.as_ref().map(Duration::as_millis) }, device_keys, token);
+request!(KeysClaimRequest from RumaKeysClaimRequest maps fields timeout { timeout.as_ref().map(Duration::as_millis) }, one_time_keys);
```

Comment:
As @turt2live mentions, and as per [docs](https://doc.rust-lang.org/stable/std/time/struct.Duration.html#method.as_millis) this returns an `u128` which is too big.